runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -133,12 +133,25 @@
 	private static final Pattern TASK_STRING_VALIDATION = Pattern.compile("[^a-zA-Z0-9' -]");
 	private static final int TASK_STRING_MAX_LENGTH = 50;
 
+	// Key names for stored task values
+	public static final String TASK_NAME_KEY = "taskName";
+	public static final String AMOUNT_KEY = "amount";
+	public static final String INIT_AMOUNT_KEY = "initialAmount";
+	public static final String TASK_LOC_KEY = "taskLocation";
+	public static final String STREAK_KEY = "streak";
+	public static final String POINTS_KEY = "points";
+	public static final String EXPEDITIOUS_CHARGES_KEY = "expeditious";
+	public static final String SLAUGHTER_CHARGES_KEY = "slaughter";
```

Comment:
These should be package-private and `@VisibleForTesting`

Reply:
Thank you, I forgot about that annotation.